lib/archethic_web/templates/layout/root.html.eex

```diff
@@ -25,7 +25,11 @@
 
         <div id="navbar" class="navbar-menu">
             <div class="navbar-end">
-    <!-- 
+
+            <a class="navbar-item" href="<%= Routes.faucet_path(@conn, :index) %>">
```

Comment:
Same, it should be displayed only when there is a configuration enabling this (not in production)

Reply:
Okay I will use the environment variable. But I also need to deploy the faucet in testnet production release.

Reply:
configuration from env variables should make it